libs/reports-list/src/lib/report-modal/report-modal.component.html

```diff
@@ -0,0 +1,8 @@
+<h1 mat-dialog-title>דיווח על חיסונים</h1>
+<div mat-dialog-content>
+  <mat-label>קופה: {{healthCareService}}</mat-label><br>
+  <mat-label>עיר: {{city}}</mat-label><br>
```

Comment:
Avoid breaking lines by <br>, it will not allow realign by CSS. Prefer alignment by CSS (you can consult others to understand how it's done properly, I'm not sure about that).

Reply:
@erezefrat 
You can use the `MatGridModule`, which allows you to easily align stuff in a grid-like manner.
Or you could use css-flex or css-grid.

Reply:
Alright, i'll look into that. thanks for the tip